javascript/channel.ts

```diff
@@ -1,6 +1,8 @@
-
+// @ts-ignore
```

Comment:
I posted a PR to recommend the Deno extension instead which resolved these errors for me, and makes intellisense stuff work too.